src/qt/radiobut.cpp

```diff
@@ -59,3 +77,44 @@ QWidget *wxRadioButton::GetHandle() const
 {
     return m_qtRadioButton;
 }
+
+void wxRadioButton::createAndJoinNewGroup( wxWindow *parent )
+{
+    m_qtButtonGroup = new QButtonGroup( parent->GetHandle() );
+    m_qtButtonGroup->addButton( m_qtRadioButton );
+}
+
+void wxRadioButton::searchForPreviousGroupToJoin( wxWindow *parent )
+{
+    wxWindowList::compatibility_iterator node = parent->GetChildren().GetLast();
+
+    for ( ; node; node = node->GetPrevious() )
+    {
+        wxWindow *child = node->GetData();
+
+        if ( wxIsKindOf( child, wxRadioButton ) )
+        {
+            if ( !child->HasFlag( wxRB_SINGLE ) )
+            {
+                QRadioButton *ptr = dynamic_cast<QRadioButton *>( child->GetHandle() );
+
+                QButtonGroup* btnGroup = ptr->group();
+                btnGroup->addButton( m_qtRadioButton );
```

Comment:
This is not correct and will lead to a crash when the first 2 radio buttons are created without any wxRB style.